src/uct/base/uct_md.h

```diff
@@ -54,9 +54,7 @@ extern ucs_config_field_t uct_md_config_rcache_table[];
  * Specific MDs extend this structure.
  */
 struct uct_md_config {
-#ifdef __cplusplus
     char __dummy;
```

Comment:
pls add comment that C++ prohibits empty structures and also let's indent this field on same column as other structs in the file